tensorflow/core/kernels/reverse_sequence_op.cc

```diff
@@ -97,7 +96,19 @@ void CheckErrors<GPUDevice>(OpKernelContext* context, int batch_dim,
                                       " vs. ", input.dim_size(batch_dim)));
 }
 
-template <typename Device, typename T>
+template <>
+void CheckErrors<GPUDevice, int32>(OpKernelContext* context, int batch_dim,
+                            int seq_dim) {
+  CheckErrorsGPU(context, batch_dim, seq_dim);
+}
+
+template <>
+void CheckErrors<GPUDevice, int64>(OpKernelContext* context, int batch_dim,
+                            int seq_dim) {
+  CheckErrorsGPU(context, batch_dim, seq_dim);
```

Comment:
Only full specialization is allowed for function templates in C++11. Partial specialization is only allowed for class templates.
http://en.cppreference.com/w/cpp/language/partial_specialization